src/main/java/htsjdk/variant/variantcontext/Allele.java

```diff
@@ -558,4 +561,11 @@ private static boolean firstIsPrefixOfSecond(final Allele a1, final Allele a2) {
         String a1String = a1.getBaseString();
         return a2.getBaseString().substring(0, a1String.length()).equals(a1String);
     }
+
+    /**
+     *  @return true if Allele is either "<NON_REF>" or "<*>"
+     */
+    public boolean isValidRefBlockAllele() {
```

Comment:
also, test this please, with a newly generated Allele, as well as the final ones